spring-cloud-dataflow-server/src/main/java/org/springframework/cloud/dataflow/server/single/Oauth2LoginEventListener.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.server.single;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.dataflow.audit.service.AuditRecordService;
+import org.springframework.cloud.dataflow.core.AuditActionType;
+import org.springframework.cloud.dataflow.core.AuditOperationType;
+import org.springframework.context.event.EventListener;
+import org.springframework.security.authentication.event.InteractiveAuthenticationSuccessEvent;
+import org.springframework.stereotype.Component;
+
+/**
+ * Event Listener class to log Login audit events.
+ *
+ * @author Deepak Gunasekaran
+ * 
+ */
+@Component
+public class Oauth2LoginEventListener {

```

Comment:
This class is in a wrong module as `spring-cloud-dataflow-server` is only used to create an executable jar. It could go under `spring-cloud-dataflow-server-core` in `org.springframework.cloud.dataflow.server.support` and bean created manually(we try to avoid component scanning) in `DataFlowServerConfiguration`.

Reply:
Class name could probably be `AuthenticationSuccessEventListener`.

Reply:
Added. Please let me know if anything else needs to be changed

Reply:
Thx, should look good now, afaik. Lemmy try to run some manual tests to test it.